### PR TITLE
add io.github.qier222.YesPlayMusic

### PR DESCRIPTION
**Pull request by NEOVALI**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
`--socket=pulseaudio` for playing music
`--device=dri` for hardware acceleration when playing music video
`--share=network` network is required to play music from [netease cloud music](https://music.163.com/)

- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:https://github.com/qier222/YesPlayMusic/issues/1738**
**and i've already contacted @qier222 on telegram**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

### CODE REVIEW

**Comment by NEOVALI** [comment]
bot, build io.gitHub.qier222.YesPlayMusic

**Comment by hfiguiere** [comment]
bot build, io.github.qier222.YesPlayMusic

File: io.github.qier222.YesPlayMusic.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+rename-icon: yesplaymusic
+rename-desktop-file: yesplaymusic.desktop
+command: yesplaymusic
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-config/fontconfig:ro
```
Comment (hfiguiere) [inline io.github.qier222.YesPlayMusic.yaml:17]: why do you need this

File: io.github.qier222.YesPlayMusic.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        sha256: 8a0776333223c0348e2cfd178b699226710f1967a776080f0bf3304c3083ba2d
+        x-checker-data:
+          is-main-source: true
+          type: json
+          url: https://api.github.com/repos/qier222/YesPlayMusic/releases/latest
+          version-query: .tag_name | sub("^v"; "")
+          url-query: '"https://github.com/qier222/YesPlayMusic/releases/download/v" + $version + "/yesplaymusic_" + $version + "_amd64.deb"'
+
+      - type: file
+        path: io.github.qier222.YesPlayMusic.appdata.xml
```
Comment (hfiguiere) [inline io.github.qier222.YesPlayMusic.yaml:65]:
```suggestion
        path: io.github.qier222.YesPlayMusic.metainfo.xml
```

File: io.github.qier222.YesPlayMusic.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      no-debuginfo: true
+    build-commands:
+      - ar x yesplaymusic.deb
+      - tar xf data.*
+      - cp -a opt/YesPlayMusic $FLATPAK_DEST/
+      - rm -Rf usr/share/icons/hicolor/1024x1024
+      - cp -a usr/share $FLATPAK_DEST/
+      - desktop-file-edit --set-key="Exec" --set-value="yesplaymusic %U" 
+          $FLATPAK_DEST/share/applications/*.desktop
+      - install -Dm644 $FLATPAK_ID.appdata.xml -t /app/share/metainfo
```
Comment (hfiguiere) [inline io.github.qier222.YesPlayMusic.yaml:36]:
```suggestion
      - install -Dm644 io.github.qier222.YesPlayMusic.metainfo.xml -t /app/share/metainfo
```

File: io.github.qier222.YesPlayMusic.yaml (hunk 1 above)
Comment (NEOVALI) [reply io.github.qier222.YesPlayMusic.yaml:17]: --socket=pulseaudio  for playing music
--device=dri    for hardware acceleration when playing music video
--share=network  network is required to play music from [netease cloud music](https://music.163.com/)
--filesystem=xdg-config/fontconfig:ro  follow user's custom font configuration

**Comment by NEOVALI** [comment]
bot, build io.github.qier222.YesPlayMusic

**Comment by NEOVALI** [comment]
bot, build io.github.qier222.YesPlayMusic

File: io.github.qier222.YesPlayMusic.yaml (hunk 1 above)
Comment (NEOVALI) [reply io.github.qier222.YesPlayMusic.yaml]: deleted xdg-config/fontconfig:ro

**Comment by NEOVALI** [comment]
bot, build io.github.qier222.YesPlayMusic

**Pull request closed without merging**